src/components-examples/material/datepicker/datepicker-dialog/datepicker-dialog-example-dialog.html

```diff
@@ -0,0 +1,10 @@
+<h2 mat-dialog-title>Calendar in a Dialog</h2>
+<mat-dialog-content align="center">
+  <div class="example-calendar-container">
+    <mat-calendar [(selected)]="data.selectedDate"></mat-calendar>
```

Comment:
This is only showing a calendar inside the dialog, not a full datepicker. Should we rename the example?

Reply:
I updated it to use the datepicker instead.